README.md

```diff
@@ -241,20 +241,51 @@ friendly** by using hashes.
 
 <h2 align="center">Contributing</h2>
 
-Most of the time, if webpack is not working correctly for you it is a simple configuration issue.
+**We want contributing to webpack to be fun, enjoyable, and educational for anyone, and everyone.** We have a [vibrant ecosystem](https://medium.com/webpack/contributors-guide/home) that spans beyond this single repo. We welcome you to check out any of the repositories in [our](http://github.com/webpack) organization or [webpack-contrib](http://github.com/webpack-contrib) which houses all of our loaders and plugins.
```

Comment:
I think "organization" should go within the `[our](...)` link e.g. `[our organization](...)`. Also maybe add "organization" after `[webpack-contrib]` as well. So with both changes:

``` md
... We welcome you to check out any of the repositories in [our organization](http://github.com/webpack) or the [webpack-contrib organization](http://github.com/webpack-contrib) which houses all of our loaders and plugins.
```